package.json

```diff
@@ -10,14 +10,86 @@
     "Other"
   ],
   "activationEvents": [
-    "onCommand:neo4j-cypher-runner.run-query"
+    "onCommand:neo4j-cypher-runner.run-query",
+    "onCommand:neo4j-cypher-runner.select-database"
   ],
   "main": "./out/extension.js",
   "contributes": {
     "commands": [
       {
         "command": "neo4j-cypher-runner.run-query",
-        "title": "Neo4j Cypher: Run Query"
+        "title": "Neo4j Cypher Runner: Run Query"
+      },
+      {
+        "command": "neo4j-cypher-runner.select-database",
+        "title": "Neo4j Cypher Runner: Select Database"
+      }
+    ],
+    "configuration": [
+      {
+        "title": "Neo4j Cypher Runner",
+        "properties": {
+          "neo4j-cypher-runner.databases": {
+            "type": "array",
+            "title": "Databases",
+            "description": "Complete functions with their parameter signature.",
+            "minItems": 1,
+            "uniqueItems": true,
+            "items": {
+              "type": "object",
+              "required": ["name", "url", "authToken"],
+              "properties": {
+                "name": {
+                  "description": "The name of database shown in the db selection",
+                  "title": "name",
+                  "type":"string"
+                },
+                "url": {
+                  "description": "The uri used to connect to the db",
+                  "title": "url",
+                  "type":"string"
+                },
+                "database": {
+                  "description": "The specific database to connect to (leave it empty to use the default db or if the Neo4j versions is previous to 4.0",
+                  "title": "database",
+                  "type":"string"
+                },
+                "authToken": {
+                  "description": "The credentials used to connect to the database",
+                  "title": "auth token",
+                  "type":"object",
+                  "required": ["scheme", "principal", "credentials"],
+                  "properties": {
+                    "scheme": {
+                      "type": "string",
+                      "description": "The used for authentication. Default: basic (means user as principal and password as credentials)",
+                      "default": "basic"
+                    },
+                    "principal": {
```

Comment:
maybe `username` instead?

Reply:
I'm using the values we have under the hood, this way the section will be generic enough for configuring any type of auth.